heft-plugins/heft-webpack5-plugin/src/WebpackConfigurationLoader.ts

```diff
@@ -36,6 +36,8 @@ export class WebpackConfigurationLoader {
     // TODO: Eventually replace this custom logic with a call to this utility in in webpack-cli:
     // https://github.com/webpack/webpack-cli/blob/next/packages/webpack-cli/lib/groups/ConfigGroup.js
 
+    const WEBPACK_CONFIG_FILENAME: string = findWebpackConfig(buildFolder, WEBPACK_CONFIG_FILENAME_REGEX);
+    const WEBPACK_DEV_CONFIG_FILENAME: string = findWebpackConfig(buildFolder, WEBPACK_CONFIG_FILENAME_REGEX);
```

Comment:
```suggestion
    const WEBPACK_CONFIG_FILENAME: string = await findWebpackConfigAsync(buildFolder, WEBPACK_CONFIG_FILENAME_REGEX);
    const WEBPACK_DEV_CONFIG_FILENAME: string = await findWebpackConfigAsync(buildFolder, WEBPACK_CONFIG_FILENAME_REGEX);
```

Reply:
This will read the folder items twice. It'd be more efficient for the function to just return both the dev and non-dev filenames.

Reply:
All for this and considered it initially. Let me make the changes locally - there's also a couple of tests failing I can look into. Once I get it updated and address any comments, I can move this from draft.

Reply:
@iclanton if we want to run this _once_ for both configs, thoughts on returning an object with keys of `dev` and `prod` in this scenario rather than an array?

Reply:
Yeah that's how I'd do it.

Reply:
Should be updated with an approach that follows that. Let me know any thoughts/feedback when you get a chance to review. Thanks @iclanton!